src/implementations/vanilla/components/mega-menu/mega-menu.scss

```diff
@@ -0,0 +1,891 @@
+/* stylelint-disable no-descending-specificity */
+
+/**
+ * Menu
+ * @define menu
+ */
+
+@use 'sass:map';
+@use '@ecl/vanilla-layout-grid/mixins/breakpoints';
+
+// Exposed variables
+$theme: null !default;
+$mega-menu: null !default;
+
+.no-scroll {
```

Comment:
I noticed that on the menu also, I'm not a big fan of this class without prefix defined in a component.
Not sure what would be the best option here, but we may put it in a more global place, to avoid duplicating it, or make it more linked to the component